pkg/standalone/standalone.go

```diff
@@ -55,6 +56,31 @@ func (h *delegateHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	w.WriteHeader(http.StatusNotFound)
 }
 
+// TODO: replace this with clientcmd
+func GetAPIServerClient(authPath string, apiServerList util.StringList) (*client.Client, error) {
```

Comment:
suggest calling this GetAPIServerClientOrDie, and hoisting the check for apiServerList < 1 outside of this function so it is called 
  if len(apiServerList) > 0 {
    cl = GetApiserverClientOrDie(...)
  } else {
    cl = nil
  }